libs/components/modals/src/lib/modules/modal/modal.component.ts

```diff
@@ -151,6 +154,8 @@ export class SkyModalComponent implements AfterViewInit, OnDestroy, OnInit {
     return this.#_ariaLabelledBy;
   }
 
+  public readonly layout = input<'none' | 'fit'>('none');
```

Comment:
Should we have a string union type to represent the possible values? This same combination is used in quite a few places.

Reply:
This is the combination that's supported by modals and the base combination for the new layout host directive. Page has a slightly different combination. For readability, I'm inclined to keep these as a simple, inline string unions rather than try to create some supertype.